packages/p2p/package.json

```diff
@@ -72,7 +82,10 @@
     "stylelint-formatter-pretty": "^1.1.3",
     "stylelint-selector-bem-pattern": "^2.1.0",
     "stylelint-webpack-plugin": "^1.1.0",
+    "terser-webpack-plugin": "^2.2.2",
     "webpack": "^4.34.0",
+    "webpack-bundle-analyzer": "^3.6.0",
     "webpack-cli": "^3.3.4"
-  }
+  },
+  "gitHead": "e17fcaa75fd7f16e5ed74f2db09c4ac9473f9fe8"
```

Comment:
what's this for?

Reply:
@oskar-binary it's indicator of last commit that was published to the npm registry. This is for cases of when commit is not package based and is git based. It is temporary and should have been cleaned. Thanks, I will remove it.